doc/psidk/src/run-infrastructure/cli/psinode.md

```diff
@@ -40,6 +40,10 @@ psinode - The psibase blockchain server
 
   Use this private key to sign blocks. Any number of keys may be provided, but only the one that matches the public key corresponding to the producer name will be used. If the correct key is not provided, then `psinode` will be unable to produce blocks.
 
+- `--pkcs11-module` *filename*
```

Comment:
I assume I can use multiple of these flags to add multiple modules?

Reply:
Yes